### PR TITLE
Add support to Dart 2.17

### PR DESCRIPTION
**Pull request by brunoocasali**
Add support to Dart 2.17

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: "3.8"
```
Comment (alallema) [inline docker-compose.yml:1]: Shouldn't this file and the addition to the contribution be in PR #164?

**Review by alallema — COMMENTED** [review]
The tests won't pass on the 2.17

**Comment by curquiza** [comment]
Looks like their are unrelated commits in this PR, regarding docker compose

File: docker-compose.yml (hunk 1 above)
Comment (brunoocasali) [reply docker-compose.yml]: Yeah it is, it pop up here because I changed the base and didn't rebase it here :p

**Review by alallema — COMMENTED** [review]
LGTM! 🎉

**Pull request merged**